cms/templates/widgets/header.html

```diff
@@ -81,6 +81,11 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Course"
                     <a href="${videos_url}">${_("Video Uploads")}</a>
                   </li>
                   % endif
+                  % if pages_and_resources_mfe_link:
+                  <li class="nav-item nav-course-courseware-videos">
```

Comment:
are we sure we are meant to be using this class `nav-course-courseware-videos` here for `Pages & Resources`?
Might be better to add a separate class for this in the `edx-platform/cms/static/sass/elements/_header.scss` file to avoid confusion/misdirection!

Reply:
even though the class does not exist in scss file, it still ok to use a dedicated class name.